lab-carolina/lib/read.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+
+module.exports = function(file){
+  var array = [];
+  fs.readFile('text1.txt', function(err, data){
+    if(err) throw err;
+    array.push(data.toString('hex', 0, 8));
+    fs.readFile('text2.txt', function(err, data){
```

Comment:
Probably want to ensure you name these parameters differently, `data` `data1`, etc. That way you ensure that inner callbacks aren't given the potential to draw on the outer scoped params.